web/.eslintrc

```diff
@@ -1,51 +1,36 @@
 {
+  // default parser does not support stage 3 so use this
+  "parser": "@babel/eslint-parser",
   "extends": [
-    "eslint:recommended",
-    "plugin:react/recommended"
+    "airbnb",
+    "prettier"
   ],
-  "parser": "@babel/eslint-parser",
-  "parserOptions": {
-    "ecmaVersion": 2017,
-    "ecmaFeatures": {
-      "jsx": true,
-      "experimentalDecorators": true,
-      "legacyDecorators": true,
-      "decoratorsBeforeExport": true
-    },
-    "sourceType": "module"
-  },
-  "env": {
-    "browser": true,
-    "node": true,
-    "es6": true
-  },
   "plugins": [
-    "react"
+    "prettier"
   ],
   "rules": {
-    "quotes": ["error", "double", { "allowTemplateLiterals": true }],
-    "indent": "off",
-    "no-console": [0],
-    "no-debugger": [0],
-    "no-unused-vars": "off",
-    "curly": [2],
-    "no-mixed-spaces-and-tabs": ["error"],
-    "jsx-quotes": ["error", "prefer-double"],
-    "react/prop-types": [0, { "skipUndeclared": true }],
-    "react/display-name": [0],
-    "react/no-unescaped-entities": [0],
-    "react/no-string-refs": [1],
-    "react/no-deprecated": [0],
-    "react/jsx-no-duplicate-props": [2],
-    "react/jsx-no-target-blank": [1],
-    "no-useless-escape": [1],
-    "template-curly-spacing": "off",
-    "no-useless-catch": "off",
-    "no-prototype-builtins": "off"
+    "prettier/prettier": [
+      "error"
+    ],
+    // use object destructured imports / exports
+    "import/prefer-default-export": "off",
```

Comment:
My take is that we don't ever use `default` exports. I think they're more confusing than just passing objects around but there might be advantages i don't know about.